BinaryConfig/contracts/interfaces/IBinaryConfig.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.10;
+
+interface IBinaryConfig {
+    function setTradingFee(uint256 newTradingFee) external;
+
+    function setClaimNoticePeriod(uint256 newNoticePeriod) external;
+
+    function setTreasury(address newTreasury) external;
+
+    function getFeeBase() external pure returns (uint256);
```

Comment:
Please order functions layout by type
Refer https://docs.soliditylang.org/en/v0.8.17/style-guide.html#order-of-layout

Recommendation Order:
 External functions
External functions that are view
External functions that are pure